devel/sparsehash/Portfile

```diff
@@ -31,16 +30,24 @@ post-patch {
     reinplace "s|@VERSION@|${version}|g" ${worksrcpath}/configure.ac
 }
 
+# While the port installs only headers, it builds tests in the process.
+# Using noarch setting breaks the build: https://github.com/macports/macports-ports/pull/15768
+installs_libs       no
```

Comment:
Per the comment that was in the Portfile that this commit removes (and also per the `long_description`), sparsehash is a header-only library, so you cannot declare that it does not install a library; it does. True, it doesn't install a dynamic library or a shared library but it does install a library of functionality implemented in headers. `installs_libs no` disables both architecture checking and license checking. We would not want the license checking to be disabled in this case; we must verify license compliance when a port uses a library.
```suggestion
```